docs/guide/ui.md

```diff
@@ -16,7 +16,7 @@ Then you can start the tests with UI by passing the `--ui` flag:
 vitest --ui
 ```
 
-Then you can visit the Vitest UI at <a href="http://localhost:51204/__vitest__/">`http://localhost:51204/__vitest__/`</a>
+Then you can visit the Vitest UI at [`http://localhost:51204/__vitest__/`](http://localhost:51204/__vitest__/)
```

Comment:
This breaks docs build

Reply:
Sorry for that, should I make other PR, other commit fixing it or force the push with the change?

Reply:
Sorry about that. Should I make other PR, commit or forze a push with the change?

Reply:
Sorry about that. I revert that change.